package.json

```diff
@@ -155,7 +155,7 @@
     "next-treat": "1.3.0",
     "nodemailer": "^6.4.13",
     "oidc-client": "^1.10.1",
-    "openapi3-ts": "^2.0.0",
+    "openapi3-ts": "2.0.0",
```

Comment:
Need to yarn install afterwards to update the lock file.

Reply:
Yup saw the failed check, was commiting updated yarn.lock